### PR TITLE
FRB data from first DSA-110 sample

### PR DESCRIPTION
**Pull request by caseyjlaw**
The main purpose of this PR is to get the FRB info from our first sample back to the main repo.
The DSA-110 fork of this repo had a few small changes that didn't get merged back in. If these other commits are not wanted, I can modify or try to extract the data-specific commit into a new PR.

### CODE REVIEW

File: frb/data/FRBs/FRB20210117A.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (profxj) [inline frb/data/FRBs/FRB20210117A.json:1]: I'm guessing this file shouldn't be here (there is one already in `FRBs`)

**Review by profxj — CHANGES_REQUESTED** [review]
One specific request.

Someone (can be me) should the build the JSON files and add those in.
I am happy to do so @caseyjlaw but it is not so easy for me to work on your fork.

If you are willing, this is the command:

`frb_build FRBs --frb TNS_NAME`

that should auto-magically put a new JSON file in `FRBs`
which you can add in and push for this PR.

The rest looks great, and I have update the Table online
which I use to build `FRBs_base.csv`.  

https://docs.google.com/spreadsheets/d/1nNwhYZWOnTcLq6Uv0KJebxMet4NzAnUKW7SFZ6n3GoY/edit?usp=sharing 

If that looks good to you,
I will update `FRBs_base.csv` eventually (in a future PR).

Thanks!

**Comment by caseyjlaw** [comment]
I'm finding that some dependencies are missing, but a key one is that `$FRB_GDB` is not defined.
If it is easier to do this with the google doc, then let's go that way. I noticed three other DSA-110 discoveries missing (from Ravi+, Sharma+, Connor+). Could you share an editable link to the doc and I'll add the info there?

**Comment by profxj** [comment]
Editor access enabled

As for `$FRB_GDB`, you can set that to any path you wish.
It's required but not used for building FRBs (only host galaxies)

**Comment by caseyjlaw** [comment]
I think the spreadsheet is updated and consistent with this PR. Do you need anything else from me?

**Comment by profxj** [comment]
One now needs to create the JSON files for each FRB
and check those in too, e.g.

https://github.com/FRBs/FRB/blob/main/frb/data/FRBs/FRB20121102A.json

This is done with a command like:

`frb_build FRBs FRB20220307B`

once you are happy with those we should be good to go.

Thx!

**Comment by caseyjlaw** [comment]
Sorry, but something still isn't right.
If I run that:

```
frb_build FRBs --frb FRB20220307B
Warning:  You need FRB/pulsars installed to use PSRCat
WARNING:  ppxf not installed
/Users/claw/anaconda3arm/envs/py38/lib/python3.8/site-packages/frb/galaxies/eazy.py:24: UserWarning: Please define the variable EAZYDIR in your environment pointing to the EAZY folder.
  warnings.warn('Please define the variable EAZYDIR in your environment pointing to the EAZY folder.')
All done!
```

Then I see no new json file.

**Comment by profxj** [comment]
Agreed.  This is what I see:

(astro) wolverine> frb_build FRBs --frb FRB20220307B
Warning:  You need FRB/pulsars installed to use PSRCat
WARNING:  ppxf not installed
You will need to install pcigale to use the cigale.py module
/home/xavier/Projects/FRB_Stuff/FRB/frb/galaxies/eazy.py:24: UserWarning: Please define the variable EAZYDIR in your environment pointing to the EAZY folder.
  warnings.warn('Please define the variable EAZYDIR in your environment pointing to the EAZY folder.')
--------------------------------------
Building FRB JSON file for FRB20220307B
Wrote data to /home/xavier/Projects/FRB_Stuff/FRB/frb/data/FRBs/FRB20220307B.json
All done!

**Comment by profxj** [comment]
@caseyjlaw -- if you prefer we could merge this in and I could
generate the JSON files fast.  I'd only ask that you review them
carefully before in that additional PR

**Comment by caseyjlaw** [comment]
Sorry, I requested a review again without making any change. The script to generate the jsons isn't working, so I updated the "base" json by hand. Is that even useful?
If not, I think we should close this and you'll need to generate all jsons yourself.

**Review by profxj — APPROVED** [review]
Pulling this in and generating the JSON files separately

**Pull request merged**